app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -630,3 +630,10 @@ def validate_hypothesis(self, value):
         if value == NimbusExperiment.HYPOTHESIS_DEFAULT.strip():
             raise serializers.ValidationError("Hypothesis cannot be the default value.")
         return value
+
+    def validate_population_percent(self, value):
+        # HACK - setting min_value=0.0001 throws a validation error with that value
+        if value < 0.000099:
```

Comment:
Can't you just change the min value in the serializer field above to that?

Reply:
Should have commented here - we can, but it changes the error message to "Ensure this value is greater than or equal to 9.9e-05" instead of `0.0001`.

Reply:
Should just be able to pass in a custom error message [like so](https://stackoverflow.com/questions/26943985/custom-error-messages-in-django-rest-framework-serializer/26975268#26975268)

Reply:
Sure, I can make that change, I mostly threw this PR up wondering if you'd point something out I was missing (I'm still not sure _why_ we need the `0.000099` solution) and it was easier to debug by seeing if it ran through a validator. I'll push that up this morning. 😎

Reply:
Pushed👌